src/ARView.js

```diff
@@ -135,7 +137,18 @@ class ARVideoRenderer {
     this.gl = gl;
     if (this.vrDisplay) {
       this.passThroughCamera = vrDisplay.getPassThroughCamera();
-      this.program = getProgram(gl, vertexSource, fragmentSource);
+      // Depending on the type of passThroughCamera, use a different
+      // target for the texture and different shaders.
+      if (this.passThroughCamera instanceof Image) {
```

Comment:
Maybe a comment indicating what's for WebARonARCore and what's for WebARonARKit+WebARonARKitSendsCameraFrames